src/commands/coverage_publish.yml

```diff
@@ -18,12 +18,20 @@ parameters:
       specified, the format will be inferred from the file
       extension or contents [possible values: simplecov, clover,
       cobertura, coverprofile, lcov, jacoco, qlty].
-    type: string
-    default: ""
+    enum:
+      - clover
+      - cobertura
+      - coverprofile
+      - jacoco
+      - lcov
+      - simplecov
+      - qlty
+    type: enum
   transform_add_prefix:
     description: |
       The prefix to add to file paths in coverage payloads,
-      to make them match the project's directory structure.
+      to make them match the project's directory structure (e.g. 
+      `/Users/distiller/project/`).
```

Comment:
I know this was out of scope.  However, I thought it could be helpful → personally, I've had to use this consistently in many projects (using the code climate reporter).

Reply:
The file paths that we need ultimately need to be relative paths from the project root, not absolute paths. (So e.g. `rails_app/app/models/user.rb`). 

The add prefix option would be used if the raw coverage data is e.g. `app/models/user.rb` and they need to be prefixed with `rails_app` in order to align with the repo structure.

So I think that this comment may be a bit confusing.

Reply:
Removed